easybuild/easyconfigs/k/Kraken/Kraken-1.1.1_CXX-CXXFLAGS.patch

```diff
@@ -0,0 +1,12 @@
+tweak Makefile such that $CXX and $CXXFLAGS are picked up from the environment
+B. Hajgato (Free University Brussels, VUB), Kenneth Hoste (HPC-UGent)
+--- kraken-1.0/src/Makefile.orig	2018-05-04 11:37:07.631634957 +0200
++++ kraken-1.0/src/Makefile	2018-05-04 11:38:07.013115388 +0200
+@@ -1,5 +1,5 @@
+-CXX = g++
+-CXXFLAGS = -Wall -fopenmp -O3
++CXX ?= g++
++CXXFLAGS ?= -Wall -fopenmp -O3
+ PROGS = db_sort set_lcas classify make_seqid_to_taxid_map db_shrink kmer_estimator
+ 
+ .PHONY: all install clean
```

Comment:
No need to create an identical duplicate patch. If old patches still work, we can use `patches = ['%(name)s-1.1_CXX-CXXFLAGS.patch']`